_pages/2019-10-14-pilot-assessment.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: "Analysis for determining suitability of M-Lab 2.0"
+permalink: /data/docs/analysis/2019-10-14-pilot-assessment/
+breadcrumb: data
+---
+
+{% jupyter_notebook "/notebooks/2019-10-14-pilot-assessment.ipynb" %}
```

Comment:
We plan to remove the `/data/docs/` section in some upcoming work, moving documentation and analysis to the `/learn` section. Given that, and though it will make a pretty long blog post, I think the notebook should just be included at the bottom of the blog post. Either way, the notebook itself has some similar formatting issues on the tables of pod names:
http://website.mlab-sandbox.measurementlab.net/data/docs/analysis/2019-10-14-pilot-assessment/index.html